idx/memory/memory_test.go

```diff
@@ -279,6 +279,30 @@ func TestDelete(t *testing.T) {
 	})
 }
 
+func TestDeleteWithLotsOfSeries(t *testing.T) {
```

Comment:
also what's the point of doing this many in a unit test anyway? doing it with 100 instead of 100k invokes all the exact same code, but runs the test in 4ms instead of 800ms on my computer.